citations/citations-impl/impl/src/java/org/sakaiproject/citation/impl/BaseCitationService.java

```diff
@@ -4096,14 +4113,12 @@ public void setAddPrefix(boolean addPrefix)
 	static
 	{
 		m_RISNoteFields.put("language", "Language: ");
-		m_RISNoteFields.put("doi", "DOI: ");
```

Comment:
Is this so that the DOI goes in the correct field?

Reply:
Yes, otherwise it is put in the notes field.